src/index.js

```diff
@@ -120,28 +131,28 @@ const createCurrentDataSet = () => {
       searchableData = bookingData;
```

Comment:
why don't you have searchableData = allData[1].bookings and roomInfo = allData[2].rooms? it might save you some redundancy and you did that on line 127

Reply:
searchableData is actually something I'm using in another event listener, so I could use it without a user attached to it.

Reply:
You could also split up this fetch a bit because it's very big. why don't you get room and bookings on load and then do all the user stuff when user/manager logs in?